### PR TITLE
Remove travis and add github actions

### PR DESCRIPTION
**Pull request by Prokyonn**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Related issues/PRs | https://github.com/sulu/sulu/issues/5693
| License | MIT

#### What's in this PR?

Switches the CI from travis to Github actions.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
         "zendframework/zendsearch": "2.*@dev",
         "symfony/monolog-bundle": "^2.8.7 || ^3.0",
         "doctrine/doctrine-bundle": "^1.4",
+        "doctrine/data-fixtures": "^1.1",
```
Comment (luca-rath) [inline composer.json:15]: why is this needed?

File: .github/workflows/test-application.yaml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+            - name: Install dependencies
+              run: |
+                  composer validate --strict
+                  composer update ${{ matrix.composer-flags }}
+
+            - name: Create database
+              run: |
+                  Tests/app/console doctrine:database:create
+                  Tests/app/console doctrine:schema:update --force
+              env: ${{ matrix.env }}
```
Comment (luca-rath) [inline .github/workflows/test-application.yaml:76]: For now, this is okay, but for the develop branch this should be done by the `bootstrap-test-environment` script

File: .github/workflows/test-application.yaml [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+            - name: Cache dependencies
+              uses: actions/cache@v2
+              id: composer-cache
+              with:
+                  path: ${{ steps.composer-cache-dir.outputs.dir }}
+                  key: ${{ runner.os }}-composer-${{ hashFiles('composer.lock') }}
+                  restore-keys: |
+                      ${{ runner.os }}-composer-
+
```
Comment (luca-rath) [inline .github/workflows/test-application.yaml:66]: Also a `lint-container` script should be added in the pr for the develop branch

File: .github/workflows/test-application.yaml [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    push:
+        branches:
+            - master
+            - develop
+            - '[0-9]+.[0-9]+'
+
+jobs:
+    test:
+        name: "Run tests with php ${{ matrix.php-version }}"
+        runs-on: ubuntu-18.04
```
Comment (luca-rath) [inline .github/workflows/test-application.yaml:14]: why not `ubuntu-latest`?

**Comment by luca-rath** [comment]
I would also replace `PrettyCI` with the `php-cs-fixer` job used in sulu.

File: .github/workflows/test-application.yaml [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+
+        strategy:
+            fail-fast: false
+            matrix:
+                include:
+                    - php-version: '5.5'
+                      composer-flags: '--prefer-lowest --prefer-dist --no-interaction'
+                      tools: 'composer:v1'
+                      env:
+                          SYMFONY__DEPRECATIONS__HELPER: disabled
```
Comment (luca-rath) [inline .github/workflows/test-application.yaml:27]: Are you sure this env variable should not be called `SYMFONY_DEPRECATIONS_HELPER`?

File: .github/workflows/test-application.yaml [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+                    - php-version: '7.4'
+                      composer-flags: '--prefer-dist --no-interaction'
+                      tools: 'composer:v2'
+                      env:
+                          SYMFONY__DEPRECATIONS__HELPER: weak
```
Comment (luca-rath) [inline .github/workflows/test-application.yaml:32]: Are you sure this env variable should not be called `SYMFONY_DEPRECATIONS_HELPER`?

File: .github/workflows/test-application.yaml (hunk 4 above)
Comment (Prokyonn) [reply .github/workflows/test-application.yaml:14]: Wasn't sure if we get any problems if they switch to ubuntu 20.04

File: .github/workflows/test-application.yaml (hunk 5 above)
Comment (Prokyonn) [reply .github/workflows/test-application.yaml:27]: Nope 😅

File: composer.json (hunk 1 above)
Comment (Prokyonn) [reply composer.json:15]: Because the `Doctrine\Common\DataFixtures\Purger\ORMPurger` is necessary in the tests and the library was removed from a dependency (I'm not sure which one it was)

File: .github/workflows/test-application.yaml (hunk 4 above)
Comment (luca-rath) [reply .github/workflows/test-application.yaml]: If I'm not wrong we used `ubuntu-latest` on the other repos too

**Review by luca-rath — APPROVED** [review]
